pubspec.yaml

```diff
@@ -83,7 +83,10 @@ dependencies:
   shared_preferences: 2.0.15 # Pinned because https://github.com/flutter/flutter/issues/118401
   slugify: ^2.0.0
   skeletons: ^0.0.3
-  swipe_to_action: ^0.2.0
+  swipe_to_action:
```

Comment:
how about to change it to https://pub.dev/packages/flutter_swipe_action_cell?

Reply:
I agree , `swipe_to_action` is out of date 2 years agooo :)))))))